app/Modules/Invoices/Controllers/InvoiceCopyController.php

```diff
@@ -71,9 +72,14 @@ public function store(InvoiceStoreRequest $request)
         }
 
         // Copy the custom fields
-        $custom = collect($fromInvoice->custom)->except('invoice_id')->toArray();
-        $toInvoice->custom->update($custom);
+        /*$custom = collect($fromInvoice->custom)->except('invoice_id')->toArray();

```

Comment:
What is happening here or what was wrong?
There should be no commented code. If it's not needed anymore, remove it.

Reply:
This is a very interesting one. Copying invoices does not work properly. When you do not have custom fields the copying does not work at all. Right now it does work, but still needs to be looked at. I've committed the removing of custom fields (was commented out) in c0e0f9f but I think we should look at this as a separate bug report